### DISCUSSION TITLE
Publishing error but latest is published

### DISCUSSION THREAD

**Discussion by hansblaauw64** [Budibase/budibase#12767] [Help] [answered]
In one project we get a published error all the time and it does not say published. But it is actually published.

It seems something in the project causes this. Is there a way to debug it?

**Comment by ConorWebb96** [comment]
Hey @hansblaauw64,

Can you provide some screenshots of the error you are seeing. Network requests as well as console logs would be very helpful. Alternative you can do a jam.dev.

- What version of Budibase are you running?
- Are you self-hosted or cloud?
- Are you using the internal Budibase db, or an external data source?
- Are you running any long automations?
- What happens if you export the app and import it again as a new app? Does this error still persist in the newly created one?

**Comment by hansblaauw64** [comment, marked as answer]
Hi,

We solved it by using chrome development tools. There was a conjob
formatting issue which caused this. It could be helpful to provide better
error messages. We also notice this when importing data, it displays failed
import but does not tell what exactly.

Cheers.

Op vr 12 jan 2024 om 13:09 schreef Conor Webb ***@***.***>:

> Hey @hansblaauw64 <https://github.com/hansblaauw64>,
>
> Can you provide some screenshots of the error you are seeing. Network
> requests as well as console logs would be very helpful. Alternative you can
> do a jam.dev.
>
>    - What version of Budibase are you running?
>    - Are you self-hosted or cloud?
>    - Are you running any long automations?
>    - What happens if you export the app and import it again as a new app?
>    Does this error still persist in the newly created one?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/12767#discussioncomment-8109446>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLZYTOZP4RXC32UBG4SZM3YOERY3AVCNFSM6AAAAABBX7K3GOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCMBZGQ2DM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Hans Blaauw

*E: ***@***.***
*W:* http://www.bluenow.nl
*T:* +31(0)6 430 44 656

**Reply by ConorWebb96** [reply]
Wonderful, glad to hear you solved this issue.

**Comment by hansblaauw64** [comment]
Hi,

Yeah good it was solved but Budibase needs better error messages :-)

Op ma 15 jan 2024 om 08:38 schreef Conor Webb ***@***.***>:

> Wonderful, glad to hear you solved this issue.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/12767#discussioncomment-8129849>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLZYTLRJKUILIIZNRERLPLYOTMH7AVCNFSM6AAAAABBX7K3GOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCMRZHA2DS>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Hans Blaauw

*E: ***@***.***
*W:* http://www.bluenow.nl
*T:* +31(0)6 430 44 656
